bedrock/mozorg/templates/mozorg/contribute/signup.html

```diff
@@ -62,6 +45,16 @@ <h1 class="page-title">{{ _('To get started, tell us what interests you.') }}</h
             </li>
           {% endfor %}
         </ul>
+
+        {% if form.errors %}
+          {{ form.non_field_errors()|safe }}
+          <ul class="errorlist indented">
```

Comment:
Yeah perhaps. The aim was for it to line up better with the icons above but, perhaps it is best to just leave it left aligned?